ethereum-consensus/src/clock.rs

```diff
@@ -195,20 +195,25 @@ impl<T: TimeProvider + Send + Sync> Clock<T> {
 
 pub type SystemClock = Clock<SystemTimeProvider>;
 
+use tokio::time::interval_at;
```

Comment:
we should also gate this
```suggestion
#[cfg(feature = "async")]
use tokio::time::interval_at;
```